package.json

```diff
@@ -69,8 +69,7 @@
     "package.json": "npx sort-package-json"
   },
   "dependencies": {
-    "@rsksmart/rif-relay-client": "1.0.0-alpha.3",
-    "@rsksmart/rif-relay-common": "github:rsksmart/rif-relay-common#feature/PP-278/PP-339/upgrade_with_ethers",
+    "@rsksmart/rif-relay-client": "github:rsksmart/rif-relay-client#PP-627/expose-client",
```

Comment:
We need to merge the [related PR](https://github.com/rsksmart/rif-relay-client/pull/67) and update the dependency here

Reply:
updated